pyproject.toml

```diff
@@ -1,15 +1,15 @@
 [project]
 name = "aggregator"
-requires-python = ">= 3.9"
-version = "0.1.3"
+requires-python = ">= 3.10"
+version = "0.3.0"
```

Comment:
Why not 0.2? Are you aligning with another thing? (no need to change to 0.2, just curious)

Reply:
there was a 0.2.1 tagged in github that didn't make it back to here 😬

Reply:
Ah - and that's harder to notice when you aren't uploading to pypi where it yells at you for trying to upload a duplicate version.